src/network/networkd-route.c

```diff
@@ -1023,3 +1038,37 @@ int config_parse_route_protocol(const char *unit,
 
         return 0;
 }
+
+int config_parse_route_type(const char *unit,
+                            const char *filename,
+                            unsigned line,
+                            const char *section,
+                            unsigned section_line,
+                            const char *lvalue,
+                            int ltype,
+                            const char *rvalue,
+                            void *data,
+                            void *userdata) {
+        Network *network = userdata;
+        _cleanup_route_free_ Route *n = NULL;
+        int r;
+
+        r = route_new_static(network, filename, section_line, &n);
+        if (r < 0)
+                return r;
+
+        if (streq(rvalue, "blackhole"))
+                n->type = RTN_BLACKHOLE;
+        else if (streq(rvalue, "unreachable"))
+                n->type = RTN_UNREACHABLE;
+        else if (streq(rvalue, "prohibit"))
+                n->type = RTN_PROHIBIT;
```

Comment:
what about RTN_UNICAST?

Reply:
Not mentioned in the RFE will add.

Reply:
Well, we don't implement RFEs 1:1 usually, but in the way they make most sense and fit the rest of the logic well. Hence, always make sure that what is implemented remains comphrensive and well-defined.

Reply:
OK